dash/resources.py

```diff
@@ -99,8 +101,3 @@ def append_script(self, script):
 
     def get_all_scripts(self, dev_bundles=False):
         return self._resources.get_all_resources(dev_bundles)
-
-    # pylint: disable=no-init, too-few-public-methods
-    class config:
-        infer_from_layout = True
-        serve_locally = False
```

Comment:
The way we did this before, despite the fact that `Scripts` was instantiated, `Scripts.config` was a singleton. Mostly this is OK, dunno if there will ever be a reason to put two `Scripts` instances in one Python session, but it caused our test cases to be coupled to each other. I had to change this to an instance when I took out all the otherwise superfluous overrides in our test cases.